components/landing/FeaturedArticles.vue

```diff
@@ -1,40 +0,0 @@
-<template>
-  <div>
-    <div class="title is-2">{{ $t('general.featuredArticlesHeading') }}</div>
-
-    <div class="flex flex-col md:flex-row gap-6 mb-5">
-      <div
-        v-for="article in articles.slice(0, 4)"
-        :key="article.id"
-        class="flex-1 last:hidden 2xl:last:visible">
-        <CardArticle
-          :link="article.web_url"
-          :image="article.thumbnail_url"
-          :title="article.title" />
-      </div>
-    </div>
-
-    <a
-      href="https://kodadotweeklyroundup.beehiiv.com/"
-      class="link"
-      target="_blank"
-      rel="nofollow noopener noreferrer">
-      {{ $t('helper.seeMore') }} >
-    </a>
-  </div>
-</template>
-
-<script lang="ts" setup>
-import { CardArticle } from '@kodadot1/brick'
-import posts from '@/script/posts.json'
```

Comment:
Let's also remove these files `script/posts.mjs` and `script/posts.json` if they are not used

Reply:
what about `CardArticle`, should also remove that ?

Reply:
<img width="837" alt="Screenshot 2024-06-09 at 12 41 37 AM" src="https://github.com/kodadot/nft-gallery/assets/39299315/5adc1b17-4bf7-4317-a783-e1215c7241a2">
@Jarsen136 i see .mjs being used here in this script, and script/posts.json is used inside mjs, do we need this script? I am not sure

Reply:
> @Jarsen136 i see .mjs being used here in this script, and script/posts.json is used inside mjs, do we need this script? I am not sure

No, please remove all of them.

Reply:
done